install/kubernetes/cilium/charts/hubble-cli/values.yaml

```diff
@@ -0,0 +1,16 @@
+# Configuration for the Hubble CLI
+image:
+  # repository of the docker image
+  repository: quay.io/cilium/hubble
+  # tag is the container image tag to use
+  tag: v0.5.1
```

Comment:
Don't we want to use the `master` branch?

Reply:
I selected `v0.5` on purpose here, so we don't accidentally break compatibility with old clients when changing the behavior of embedded Hubble.

But you bring up a good point with regard to version testing. The way I see the use of Hubble CLI versions in the CI going forward:

1. Have embedded Hubble enabled for most of the e2e tests. Use a known-to-work recent Hubble CLI version to dump the flow log at the end of a test and where possible assert things about its expected output.
2. Have some dedicated tests that test that supported versions of the Hubble CLI (e.g. `v0.5`) still work as expected (whatever that means for specific features).

The tests added by this PR are in the second category. I plan follow-up PRs for point 1), there I think it makes sense to use a more recent version if possible.

On a side note: I don't think you proposed this, but I'm vary on directly tagging a master branch (i.e. `latest`), as merges on `cilium/hubble` are not blocked by Cilium CI, but would have the potential to break CI for all of Cilium if we used `latest`. Therefore I think we want to introduce docker image tags for "known-to-work" master commits that we bump regularly.